sql/hive/src/test/scala/org/apache/spark/sql/hive/MultiDatabaseSuite.scala

```diff
@@ -297,7 +297,7 @@ class MultiDatabaseSuite extends QueryTest with SQLTestUtils with TestHiveSingle
             |)
             """.stripMargin)
         }.getMessage
-        assert(message.contains("`t:a` is not a valid name for tables/databases. " +
+        assert(message.contains("`t:a` is not a valid name for tables/schemas. " +
```

Comment:
@srielau Since you are here, could you use `checkError` (and the error class), please.

Reply:
**Hehe** Yes, I had trouble overcoming my inner pig.

Reply:
Done